### PR TITLE
track improvements towards pep8 compliance

### PR DESCRIPTION
**Pull request by JessicaS11**
After updating the GH actions (#475), in order to get the checks to pass we needed to add a lot of ignore codes to the .flake8 config file. This draft PR removes those ignores so PRs moving forward have a spot to look to see if any edited files are out of compliance (but the checks will still pass as we work towards compliance for legacy code).

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -12,29 +12,29 @@ per-file-ignores =
...
+;     # comparison syntax
+;     E712
+;     # comparison syntax in tests
+;     E714
+;     # comparison syntax in tests
+;     E721
+;     # bare except
+;     E722
+;     # ambiguous var name
+;     E741
```
Comment (JessicaS11) [inline .flake8:29]:
```suggestion
```

File: .flake8 [hunk 2]
```diff
@@ -12,29 +12,29 @@ per-file-ignores =
...
+;     # imported but unused
+;     F401
+;     # unable to detect undefined names
+;     F403
+;     # assigned and unused (in tests)
+;     F841
+;     # line break before binary operator
+;     W503
```
Comment (JessicaS11) [inline .flake8:37]:
```suggestion
```

**Comment by JessicaS11** [comment]
Closing as this workflow was transitioned to using ruff #543 (and #571 and others).

**Comment by mfisher87** [comment]
Thanks to @cclauss we're pretty darn compliant now! :tada:

**Pull request closed without merging**